src/EnergyPlus/Plant/Enums.hh

```diff
@@ -189,6 +189,15 @@ namespace EnergyPlus::DataPlant {
         DualOp,    // Constant for Cooling or Heating Operation
     };
 
+    // branch loop type for absorption chillerheater models
+    enum class BrLoopType
+    {
+        Chiller,
+        Heater,
+        Condenser,
+        NoMatch
+    };
```

Comment:
Yup, great!  In a future set of changes the actual type name might be tailored a little more to the actual application, but this is fine for now.

Reply:
@Myoldmopar Thanks! Yes, the warnings are usually recurring. So it makes sense to condense them.